lib/src/models/view/schedule_search.dart

```diff
@@ -0,0 +1,36 @@
+import "package:ramaz/data.dart";
+import "package:ramaz/models.dart";
+
+extension on Subject{
+  String get id => "";
+}
```

Comment:
```suggestion

```

This was a small "hack" to stop the error. But we do need to actually properly fix this. I'll let Brayden know

Reply:
yeah i know i just didnt want to get an error when i pushed